aleks/resume.md

```diff
@@ -1,29 +1,46 @@
-## Résumé: Aleksandar Todorović 	
+# Résumé: Aleksandar Todorović 	
 
 ### Personal Details:
 
 #### Affiliation:
-Bits of Freedom, Open Web Fellow
+[Bits of Freedom](https://www.bof.nl/), Open Web Fellow
 
 #### Contact:
-* **Email**: [aleksandar@mozillafoundation.org](mailto:aleksandar@mozillafoundation.org)
-* **Twitter**: [@r3bl_](https://twitter.com/r3bl_)
-* **GitHub**: [@aleksandar-todorovic](https://github.com/aleksandar-todorovic)
 
-As part of the 2017 Open Web Fellows program, I completed/participated in the following:
+* **Emails**: [aleksandar@mozillafoundation.org](mailto:aleksandar@mozillafoundation.org) / [aleksandar@r3bl.me](mailto:aleksandar@r3bl.me) / [aleksandar.todorovic@bof.nl](mailto:aleksandar.todorovic@bof.nl)
+* **Microblogging**: [@r3bl_](https://twitter.com/r3bl_) on Twitter, [@r3bl@mastodon.social](https://mastodon.social/@r3bl) on ActivityPub/OStatus fediverse. 
+
+#### Other useful links:
+
+* **Personal website:** [r3bl.me](https://r3bl.me)
+* **Personal blog:** [blog.r3bl.me](https://blog.r3bl.me)
+* **GitHub**: [@aleksandar-todorovic](https://github.com/aleksandar-todorovic) for smaller, personal projects / [@r3bl](https://github.com/r3bl) for publicly available web projects
+
+As part of the 2017 Open Web Fellows program, I completed/participated/worked on the following:
 
 #### EVENTS & TALKS
 
 Event | Location | Date | Role
 ----- | -------- | ---- | -----
-[Cool Event](URL) | Some place, Rad country | Sept 2017 | Speaker  
-
+Mozilla Festival | London, UK | 
```

Comment:
`[Mozilla Festival](https://mozillafestival.org/) | London, UK | Oct 2017 | Fellow`

Reply:
Ah, this was an idea that I forgot to delete before creating the event.

Is it worth including MozFest and IFF or should I just leave this subheading empty?

Reply:
Yes why not, you participated, I’m sad you probably won’t include your CCC talk, but you might do that too

Reply:
^ Please reply to the edit above before I commit the changes you've requested.